### PR TITLE
feat(python): Add stack information to gRPC errors

### PR DESCRIPTION
**Pull request by tebeka**
Fixes ENG-1895

### CODE REVIEW

File: runtimes/pythonrt/runner/main.py [hunk 1]
```diff
@@ -110,6 +110,14 @@ def killIfStartWasntCalled(runner):
         os._exit(1)
 
 
+def abort_with_exception(context, status, err):
+    io = StringIO()
+    print(err, file=io)
```
Comment (itayd) [inline runtimes/pythonrt/runner/main.py:114]: this wont be going to the print hijacking you did?

Comment (tebeka) [reply runtimes/pythonrt/runner/main.py:114]: It will, so the user will be able to see it.

**Review by daabr — APPROVED** [review]
Not sure what @itayd means in his question (i.e. if there's some concern there), but I'm personally waiting for this

**Comment by itayd** [comment]
> Not sure what @itayd means in his question (i.e. if there's some concern there), but I'm personally waiting for this

he did a print method hijacking to capture all logs. i wonder if it'll go there or some kind of unified error.

**Pull request merged**